mon-pix/app/components/challenge/statement/tooltip.js

```diff
@@ -68,12 +67,18 @@ export default class Tooltip extends Component {
   }
 
   @action
-  confirmInformationIsRead() {
+  async confirmInformationIsRead() {
+    await this._rememberUserHasSeenChallengeTooltip();
     this.shouldDisplayTooltip = false;
-    this._notifyChallengeTooltipIsClosed();
   }
 
-  _notifyChallengeTooltipIsClosed() {
-    this.args.onTooltipClose();
+  async _rememberUserHasSeenChallengeTooltip() {
+    if (this.currentUser.user) {
+      if (this.args.challenge.focused && !this.currentUser.user.hasSeenFocusedChallengeTooltip) {
+        await this.currentUser.user.save({ adapterOptions: { tooltipChallengeType: 'focused' } });
+      } else if (!this.args.challenge.focused && !this.currentUser.user.hasSeenOtherChallengesTooltip) {
+        await this.currentUser.user.save({ adapterOptions: { tooltipChallengeType: 'other' } });
+      }
+    }
```

Comment:
J'aime bien mettre un "shortcut" quand il y a un `if` qui englobe tout comme ça, ça enlève un peu d'indentation :

```suggestion
    if (!this.currentUser.user) return

    if (this.args.challenge.focused && !this.currentUser.user.hasSeenFocusedChallengeTooltip) {
      await this.currentUser.user.save({ adapterOptions: { tooltipChallengeType: 'focused' } });
    } else if (!this.args.challenge.focused && !this.currentUser.user.hasSeenOtherChallengesTooltip) {
      await this.currentUser.user.save({ adapterOptions: { tooltipChallengeType: 'other' } });
    }
```